### PR TITLE
Update AWS monitoring docs

### PR DESCRIPTION
**Pull request by mohabusama**
- Add latest docs
- Update missing parts in dcos

### CODE REVIEW

File: docs/user/monitoringonaws.rst [hunk 1]
```diff
@@ -37,27 +51,153 @@ Similarly, you can pull CPU-related metrics from CloudWatch. The Prometheus Node
...
+    # Classic ELB
+    lb_name = entity['name']
+    key = 'LoadBalancerName'
+
+    # Check if Application ELBv2 entity
+    if entity.get('elb_type') == 'application':
+        lb_name = entity['cloudwatch_name']
+        key = 'LoadBalancer'
+
+    cloudwatch().query_one({key: lb_name}, 'RequestCount', 'Sum', 'AWS/ELB')
```
Comment (hjacobs) [inline docs/user/monitoringonaws.rst]: the namespace is also different for ELBv2

Comment (mohabusama) [reply docs/user/monitoringonaws.rst]: True! Will fix it.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**